.github/workflows/cd-push-docker-mysql.yaml

```diff
@@ -0,0 +1,30 @@
+name: Push mysql quick boot images to Docker Hub
+on:
+  push:
+    paths:
+      - ops/mysql/**
+    branches:
+      - master
+
+jobs:
+  push-docker:
+    strategy:
+      matrix:
+        dialect: [ "mysql:latest", "mysql:5.6.35", "mysql:5.7.26", "mysql:8", "mariadb:latest", "mariadb:10.7", "mariadb:10.2.32", "mariadb:10.3.13" ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build & Push ${{ matrix.dialect }} Docker Image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./ops/mysql/Dockerfile
+          push: true
+          tags: arigaio/${{ matrix.dialect }}-${{ github.sha }},arigaio/${{ matrix.dialect }}-latest
```

Comment:
I dont think you have a valid image tag here. Double : in its name

Reply:
what do you suggest?
just arigaio/${{ matrix.dialect }} ?

Reply:
We should add a prefix. `arigaio/eph-${{ matrix.dialect }}`. Or we need to separate the dialect and its version?

Reply:
I prefer `arigaio/mysql:<tag>`.